app/views/games/show.html.erb

```diff
@@ -3,7 +3,8 @@
 <br>
 
```

Comment:
It might be nice in the future to add html that tells the user what player they are: either black or white, since the board orientation is the same.